lib/main.dart

```diff
@@ -24,22 +24,25 @@ import 'services/services.dart';
 import 'utils/utils.dart';
 import 'widgets/widgets.dart';
 
-final AtSignLogger _logger = AtSignLogger(AtEnv.appNamespace);
+AtSignLogger _logger = AtSignLogger(AtEnv.appNamespace);
```

Comment:
This is going to cause the same issue, since AtEnv.appNamespace is still being called before AtEnv.load(). Instead of passing AtEnv.appNamespace, pass a String.

Reply:
That's why called it a second time in line 45 after AtEnv.load() is called

Reply:
fixed